docs/scripts/gen_popular_issues.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from datetime import datetime
+
+import pandas as pd
+import requests
+import mkdocs_gen_files
+
+
+REPOSITORY = "argilla-io/distilabel"
+DATA_PATH = "sections/community/popular_issues.md"
+
+GITHUB_ACCESS_TOKEN = os.environ[
+    "GH_ACCESS_TOKEN"
+]  # public_repo and read:org scopes are required
```

Comment:
Can we change this to `os.getenv("GH_ACCESS_TOKEN")` and only fetch the data from GitHub if is not `None`? It's a bit cumbersome to have to set a `GH_ACCESS_TOKEN` locally.

Reply:
Also add `pandas` as a explicit dependency in the `docs` extra.